File: 프로그래머스/0/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class Solution {
    public string[] solution(string myString) {
        string[] answer = myString.Split('x',StringSplitOptions.RemoveEmptyEntries);
        List<string> arr = answer.ToList<string>();
        arr.Sort();
        return arr.ToArray();
    }
}

File: 프로그래머스/0/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>
using namespace std;

vector<string> solution(string myString) {
    vector<string> answer;
    stringstream ss(myString);
    string temp;
    
    while(getline(ss,temp,'x'))
    {
        if(!temp.empty())
            answer.push_back(temp);
    }
    
    std::sort(answer.begin(), answer.end());
    return answer;
}